.github/workflows/ci.yml

```diff
@@ -54,21 +45,41 @@ jobs:
       run: |
         echo $TERM
         # Tests for when using polling
-        julia --project --code-coverage=user -e 'ENV["JULIA_REVISE_POLL"]="1"; using Pkg, Revise; include(joinpath(dirname(pathof(Revise)), "..", "test", "polling.jl"))'
+        julia --project --code-coverage=user -e '
+          ENV["JULIA_REVISE_POLL"]="1"
+          using Pkg, Revise
+          include(joinpath(dirname(pathof(Revise)), "..", "test", "polling.jl"))
+        '
         # The REPL wasn't initialized, so the "Methods at REPL" tests didn't run. Pick those up now.
-        TERM="xterm" julia --project --code-coverage=user -e 'using InteractiveUtils, REPL, Revise; @async(Base.run_main_repl(true, true, false, true, false)); while !isdefined(Base, :active_repl_backend) sleep(0.1) end; pushfirst!(Base.active_repl_backend.ast_transforms, Revise.revise_first); include(joinpath("test", "runtests.jl")); REPL.eval_user_input(:(exit()), Base.active_repl_backend)' "Methods at REPL"
+        TERM="xterm" julia --project --code-coverage=user -e '
+          using InteractiveUtils, REPL, Revise
```

Comment:
In bash, single (strong) quotes can span multiple lines, this is clearer than a single line statement with semicolons.